### PR TITLE
Filebrowser ui redesign

### PR DESCRIPTION
**Pull request by haidian6672**
new-icon

### CODE REVIEW

File: src/filebrowser/file-browser-dialog.cpp [hunk 1]
```diff
@@ -257,7 +258,9 @@ void FileBrowserDialog::createToolBar()
     connect(forward_button_, SIGNAL(clicked()), this, SLOT(goForward()));
 
     gohome_action_ = new QAction(tr("Home"), this);
-    gohome_action_->setIcon(QIcon(":images/filebrowser/home.png"));
+    QIcon home_icon;
+    home_icon.addFile(":images/filebrowser/home.png", QSize(16, 16));
+    gohome_action_->setIcon(home_icon);
```
Comment (lins05) [inline src/filebrowser/file-browser-dialog.cpp]: Why not simply  `QIcon home_icon(":images/filebrowser/home.png")` ?

File: src/filebrowser/file-browser-dialog.h [hunk 2]
```diff
@@ -171,16 +174,18 @@ private slots:
...
-    QToolBar *status_bar_;
+    QWidget *status_bar_;
+    QHBoxLayout *status_layout_;
     QToolButton *upload_button_;
     QMenu *upload_menu_;
     QAction *upload_file_action_;
     QAction *upload_directory_action_;
     QAction *mkdir_action_;
     QLabel *details_label_;
+    QLabel *refresh_label_;
```
Comment (lins05) [inline src/filebrowser/file-browser-dialog.h]: Seems `refresh_label_` is never used?

**Comment by lins05** [comment]
Mostly LGTM. One minor ui issue: when the forward/backward arrows are hovered, there would be some extra white background. @freeplant Do you think we need to change it?

screen shot 2017-08-02 at 4 44 47 pm

**Comment by freeplant** [comment]
Yes, it need to be improved.

**Pull request merged**